lib/model/frames.js

```diff
@@ -75,7 +76,56 @@ const Comment = Frame.define(
   embedded('actor')
 );
 
-const Config = Frame.define(table('configs'), 'key', 'value', 'setAt', species('config'));
+// configKeys holds options for each config. It determines whether the config
+// can be directly set over the API, and if so, how. It also determines
+// whether/how the config value is returned over the API and saved in the audit
+// log. (Using Object.create(null) to prevent access to Object.prototype
+// properties.)
+const configKeys = Object.assign(Object.create(null), {
```

Comment:
i think this is a fine approach. my two thoughts are
1 maybe this is heavy enough to move to its own `frames/` file. any time these things sprout too much logic i feel like this file doesn't want it.
2 i wonder if there is a way to encode these different capsules as separate frame types or subtypes rather than with sort of this internal guts way? feels like it would be more confluent w the other declarations and probably keep these overrides from becoming messy